bindings/rust/aws-lc-sys-template/build.rs

```diff
@@ -0,0 +1,271 @@
+/* Copyright (c) 2022, Google Inc.
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY
+ * SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION
+ * OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR IN
+ * CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE. */
+
+// SPDX-License-Identifier: Apache-2.0
+// Modifications Copyright Amazon.com, Inc. or its affiliates. See GitHub history for details.
+
+use regex::Regex;
+use std::{env, fs, io};
+use std::fs::File;
+use std::io::{BufRead, BufReader, BufWriter, Write};
+use std::path::{Path, PathBuf};
+use std::process::Command;
+
+fn modify_bindings(bindings_path: &PathBuf, prefix: &str) -> io::Result<()> {
```

Comment:
The current logic here is not sufficient -- it only updates the function symbols, but there are other symbols that still need to be modified. For example, the following can currently be found in bindings.rs:
```
...
extern "C" {
    pub static aws_lc_0_1_0_ASN1_BMPSTRING_it: ASN1_ITEM;
}
extern "C" {
    pub static aws_lc_0_1_0_ASN1_GENERALSTRING_it: ASN1_ITEM;
}
extern "C" {
    pub static aws_lc_0_1_0_ASN1_IA5STRING_it: ASN1_ITEM;
}
...
```